Configurations/commands.js

```diff
@@ -672,7 +672,7 @@ const commands = {
 			category: `Utility 🔦`,
 		},
 		room: {
-			usage: `"text" or "voice" or "both" [<user 1>] [|] [<user 2>] [|] [...]`,
+			usage: `"text" or "voice" [<user 1>] [<user 2>] [...]`,

```

Comment:
Why? I still want the both one. Which creates a combo of voice and text channels... Will be super useful for gamers or anyone needing a quick double-channel

Reply:
It was removed because it wasn't implemented in the original source. We're tight on schedule so I'm prioritizing porting the base features of these commands over improving or expanding them.

Reply:
If you have time, feel free to implement it yourself though.